dist/tools/nacl_key_generator/generate-nacl-keys.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
I'd prefer a more generic toolset. One to create a keypair. use xxd to convert that to a .h file usable by code.